tools/data/sthv2/preparing_sthv2.md

```diff
@@ -9,7 +9,7 @@ First of all, you have to sign in and download annotations to `$MMACTION2/data/s
 
 ## Step 2. Prepare Videos
 
-Then, you can download all data parts to `$MMACTION2/data/sthv2/` and use the following command to extract.
+Then, you can download all data parts to `$MMACTION2/data/sthv2/` and use the following command to unzip.
```

Comment:
the file format is not zipped. so unzip is not exact

Reply:
It may be better to use "uncompress".

Reply:
extract is okay.